index.js

```diff
@@ -1,48 +1,66 @@
+  let userHours = document.querySelector("#hoursPerWeek").value;
+  console.log(userHours);
+
+  // Validate the user's input - check with ifs. display errors by showing hidden text
+  validateData(userEmail, userLevel, userHours);
+
 });
 
-let userEmail;
-let userLevel;
-let userHours;
-
-console.log(userEmail);
-console.log(userLevel);
-console.log(userHours);
-
-// Validate the user's input - check with ifs. display errors by showing hidden text
-//  Check if the user has provided an email address - "E-mail address is required" is it an actual email
-//  Check if the user has selected a level - "You must select a course level"
-//  Check if the user has specified at least one hour of study - "You must select at least one hour per week of lessons"
-//  Check if the number of hours requested is within the allowed range - "You can only have a maximum of 5 hours per week with the beginner plan" / "You can only have a maximum of 10 hours per week with the advanced plan"
-function ValidateData() {
-  if (userEmail == "") {
+
+function validateData(email, level, hours) {
+  let emailError = document.getElementById("email_error");
+  let levelError = document.getElementById("level_error");
+  let hoursError = document.getElementById("hours_error");
+  //  Check if the user has provided an email address - "E-mail address is required" is it an actual email
+  if (email == "") {
+    emailError.textContent = "You must provide a valid email";
     console.log("no email provided");
+  } else {
+    emailError.textContent = "";
   }
 
-  if (userLevel == "") {
+  //  Check if the user has selected a level - "You must select a course level"
+  if (level == "") {
+    levelError.textContent = "You must select a level";
     console.log("no level provided");
+  } else {
+    levelError.textContent = "";
   }
 
-  if (userHours < 1) {
+  //  Check if the user has specified at least one hour of study - "You must select at least one hour per week of lessons"
+  if (hours < 1) {
+    hoursError.textContent = "You must sign up for at least one hour per week";
     console.log("hour too low");
-  }
-
-  if (userLevel == "basic" && userHours > 5) {
-    console.log("beginner hours to high");
-  }
-
-  if (userLevel == "advanced" && userHours > 10) {
-    console.log("advanced hours to high");
+    
+    //  Check if the number of hours requested is within the allowed range - "You can only have a maximum of 5 hours per week with the beginner plan" / "You can only have a maximum of 10 hours per week with the advanced plan"
+  } else if (level == "basic" && hours > 5) {
+    hoursError.textContent = "You can only have a maximum of 5 hours per week with the basic plan";
+    console.log("beginner hours too high");
+  } else if (level == "advanced" && hours > 10) {
+    hoursError.textContent = "You can only have a maximum of 10 hours per week with the advanced plan";
+    console.log("advanced hours too high");
+  } else {
+    hoursError.textContent = "";
```

Comment:
Excellent work. You are displaying error messages successfully to the user.

You just need to [style the error messages fully](https://gedsargent.github.io/code-monster/guides/programming/harmony-music/error-handling/#display-error-messages)

It looks as though you are ready to start the tutorial on [refactoring your code](https://gedsargent.github.io/code-monster/guides/programming/harmony-music/code-refactor/).